src/main/java/Project_16x16/App.java

```diff
@@ -20,22 +20,22 @@
 import processing.event.MouseEvent;
 import processing.javafx.PSurfaceFX;
 
-import scene.PScene;
-import scene.GameplayScene;
-import scene.MainMenu;
-import scene.PauseMenu;
+import Project_16x16.scene.PScene;
+import Project_16x16.scene.GameplayScene;
+import Project_16x16.scene.MainMenu;
+import Project_16x16.scene.PauseMenu;
 
 /**
- * <h1>SideScroller Class</h1>
+ * <h1>App Class</h1>
  * <p>
- * The SideScroller class is the main class. It extends the processing applet,
+ * The App class is the main class. It extends the processing applet,
  * and is the heart of the game.
  * </p>
  */
-public class SideScroller extends PApplet {
+public class App extends PApplet {
```

Comment:
If we want to rename the main class, it should be done in another PR to keep changes in this PR relevant to maven only.